doc/release/0.19.0-notes.rst

```diff
@@ -54,6 +54,9 @@ compute the coeficients of a second-order IIR peak (resonant) filter.
 The function `scipy.signal.minimum_phase` was added to convert linear-phase
 FIR filters to minimum phase.
 
+The functions `scipy.signal.upfirdn` and `scipy.signal.resample_poly` are now
+substantially faster when operating on n-dimensional arrays when n > 1.
```

Comment:
Let's not get people's hopes up too high, probably "when operating on some n-dimensional" or elaborate a bit more on when it's useful, i.e., people with a `10x100000` array shouldn't get too excited for `axis=1` but should for `axis=0`.